.pre-commit-config.yaml

```diff
@@ -66,8 +66,8 @@ repos:
     -   id: changelogs-rst
         name: changelog filenames
         language: fail
-        entry: 'changelog files must be named ####.(breaking|bugfix|deprecation|doc|feature|improvement|trivial|vendor).rst'
-        exclude: changelog/(\d+\.(breaking|bugfix|deprecation|doc|feature|improvement|trivial|vendor).rst|README.rst|_template.rst)
+        entry: 'changelog files must be named ####.(breaking|bugfix|contrib|deprecation|doc|feature|improvement|misc|packaging|vendor).rst'
+        exclude: changelog/(\d+\.(breaking|bugfix|contrib|deprecation|doc|feature|improvement|misc|packaging|vendor).rst|README.rst|_template.rst)
```

Comment:
this one is missing the .gitignore

Reply:
@RonnyPfannschmidt fixed! Marking this thread as resolved.